### PR TITLE
fix: add WillPopScope to symptom checker

### PR DESCRIPTION
**Pull request by britannio**
Closes https://github.com/WorldHealthOrganization/app/issues/1295

Tapping the back button or pressing the back button on Android will now transition to the previous options page and going back from the results page will bring you back to the home page.

---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

File: client/flutter/lib/pages/symptom_checker/symptom_checker_view.dart [hunk 1]
```diff
@@ -133,15 +154,17 @@ class _SymptomCheckerViewState extends State<SymptomCheckerView>
     _model.answerQuestion(answerIds);
     if (!_model.isFatalError && _model.results != null) {
       Navigator.of(context)
-          .pushNamed('/symptom-checker-results', arguments: _model);
+          .pushReplacementNamed('/symptom-checker-results', arguments: _model);
```
Comment (advayDev1) [inline client/flutter/lib/pages/symptom_checker/symptom_checker_view.dart:157]: I had it this way before, but we want the back on results to go to the last Q.

Comment (britannio) [reply client/flutter/lib/pages/symptom_checker/symptom_checker_view.dart:157]: @advayDev1  The only way to escape would then be to go back through every single question.

Comment (advayDev1) [reply client/flutter/lib/pages/symptom_checker/symptom_checker_view.dart:157]: No.  You press the 'Done' button?

Comment (advayDev1) [reply client/flutter/lib/pages/symptom_checker/symptom_checker_view.dart:157]: I am approving and we can fix this in the next PR.

**Pull request merged**